account_financial_risk/security/security.xml

```diff
@@ -4,17 +4,22 @@
         <field name="name">Financial Risk</field>
         <field name="sequence" eval="99" />
     </record>
+    <record id="privilege_financial_risk" model="res.groups.privilege">
+        <field name="name">Financial Risk Privilege</field>
```

Comment:
The name shouldn't be this one, as it's weird (remove the ` Privilege` part):

<img width="668" height="151" alt="image" src="https://github.com/user-attachments/assets/14eb39cf-fbe1-4492-9138-8284d912ec25" />

And in any case, I think it should be inside "Accounting" category, don't you think?